dsl/src/commonMain/kotlin/it/unibo/collektive/field/operations/NumberUtils.kt

```diff
@@ -0,0 +1,66 @@
+package it.unibo.collektive.field.operations
```

Comment:
wait aren't these going to be autogenerated?

Reply:
Yes, but at the moment the code generation is stuck.
We can consider to drop this file as soon as the codegen module is ready and merged

Reply:
Let us first merge the codegen, then, it will simplify and reduce the boilerplate

Reply:
Codegen is in now. Let us drop this one.